ScoreSaber/Core/Services/LeaderboardService.cs

```diff
@@ -45,7 +52,11 @@ public async Task<Leaderboard> GetCurrentLeaderboard(BeatmapKey beatmapKey) {
             return null;
         }
       
-        private string GetLeaderboardUrl(BeatmapKey beatmapKey, PlatformLeaderboardsModel.ScoresScope scope, int page, bool filterAroundCountry) {
+        private string GetLeaderboardUrl(BeatmapKey beatmapKey, ScoreSaberLeaderboardViewController.ScoreSaberScoresScope scope, int page) {
+            
+            if(!beatmapKey.levelId.Contains("custom_level_")) {
+                return null;
+            }
```

Comment:
Ah, didnt know about that one. Thats good to know. Will fix. Will wait for umbra to say something before resolving this one

Reply:
On closer inspection, its a beatmapkey, so we cant check if it has precalculated data here

Reply:
Also been talking to @Umbranoxio on Discord: we will be enabling the leaderboards for OST/DLC

Reply:
Made it allow ost and dlc, cant find leaderboards right now ofc.

Reply:
Actually you should be able to find leaderboards, since we support ost leaderboards on quest, we just have special hashes for them. [here is the implementation](https://github.com/ScoreSaber/scoresaber-quest/blob/bc165ef41d3b1a0505ea65050ee69bce5b684bbe/src/Utils/StringUtils.cpp#L31)

Allthough SongCore may differ from Quest. Ensure the output of the hash looks like this: `ost_CycleHit`